mobile/src/lib/crash-reporter.js

```diff
@@ -0,0 +1,15 @@
+/* global __SENTRY_TOKEN__ */
+import Raven from 'raven-js'
+
+export function logException (err, context) {
+  if (!Raven.isSetup()) {
+    Raven.config(`https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`).install()
```

Comment:
You do approve this PR, but you suggest to add a new webpack parameter for `project ID`. What should I do?